src/Compilers/Core/Portable/Operations/IConversionExpression.cs

```diff
@@ -17,14 +17,31 @@ public interface IConversionExpression : IHasOperatorMethodExpression
         /// Value to be converted.
         /// </summary>
         IOperation Operand { get; }
+
+#pragma warning disable RS0010 // Avoid using cref tags with a prefix
         /// <summary>
-        /// Kind of conversion.
+        /// Gets the underlying common conversion information.
         /// </summary>
-        ConversionKind ConversionKind { get; }
+        /// <remarks>
+        /// If you need conversion information that is language specific, use either
+        /// <see cref="T:Microsoft.CodeAnalysis.CSharp.IConversionExpressionExtensions.GetConversion(IConversionExpression)"/> or
+        /// <see cref="T:Microsoft.CodeAnalysis.VisualBasic.GetConversion(IConversionExpression)"/>.
+        /// </remarks>
+#pragma warning restore RS0010 // Avoid using cref tags with a prefix
+        CommonConversion Conversion { get; }
         /// <summary>
         /// True if and only if the conversion is indicated explicity by a cast operation in the source code.
         /// </summary>
-        bool IsExplicit { get; }
+        bool IsExplicitInCode { get; }
+        /// <summary>
+        /// True if the conversion will fail with a <see cref="InvalidCastException"/> at runtime if the cast fails. This is false for C#'s
+        /// <code>as</code> operator and for VB's <code>TryCast</code> operator.
```

Comment:
it should also be false for thigns like a numeric conversion in an unchecked context.  ensure we test this.